spaceship/spec/portal/app_spec.rb

```diff
@@ -124,7 +124,7 @@
     it 'updates the name of the app by given bundle_id' do
       stub_request(:post, "https://developer.apple.com/services-account/QH65B2/account/ios/identifiers/updateAppIdName.action").
         with(body: { "appIdId" => "B7JBD8LHAA", "name" => "The New Name", "teamId" => "XXXXXXXXXX" }).
-        to_return(status: 200, body: JSON.parse(PortalStubbing.adp_read_fixture_file('updateAppIdName.action.json')), headers: {})
+        to_return(status: 200, body: PortalStubbing.adp_read_fixture_file('updateAppIdName.action.json'), headers: { 'Content-Type' => 'application/json' })
```

Comment:
That's Ruby 2.0 syntax we use everywhere in the fastlane code base, including the `Fastfile`